src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java

```diff
@@ -471,5 +515,6 @@ public void copyFiltersToScriptTest() throws Exception {
         mvc.perform(get("/" + VERSION + "/filters-contingency-lists/tic")
                 .contentType(APPLICATION_JSON))
                 .andExpect(status().isOk());
+        cleanDB();
```

Comment:
useless because it's already in the setup

Reply:
Done